addon/tailwind/components/bourbon-table.css

```diff
@@ -0,0 +1,49 @@
+.BourbonTable {
+  height: 224px;
+  width: 100%;
+  table-layout: auto;
+}
+
+.BourbonTable-header {
+  color: #9EA0A5;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  text-align: left;
+
+  min-height: 44px;
+}
+
+.BourbonTable-row {
+  font-size: 14px;
+  line-height: 18px;
+  text-transform: capitalize;
+  text-align: left;
+
+  height: 90px;
+  background-color: rgba(255,255,255,0);
+  box-shadow: 0 1px 0 0 #EAEDF3;
+}
+
+.BourbonTable-row:first-child {
+  height: 44px;
+}
+
+.BourbonTable-rowItem {
+  color: #6B6C6F;
+}
+
+.BourbonTable-rowItem:first-child {
+  color: #000;
+  font-weight: 500;
+}
+
+.BourbonTable-rowItem:last-child * {
+  float: right;
+}
```

Comment:
What effect is produced by `float: right`-ing all children of the last row? My CSS isn't sharp enough to visualize it.

Reply:
It is basically making the last item hug the end of the row.  Same thing as `text-align: right`.  We wanted to create space between the other items and the last item in the row.